stdlib/Serialization/src/Serialization.jl

```diff
@@ -520,11 +520,7 @@ function serialize_typename(s::AbstractSerializer, t::Core.TypeName)
         serialize(s, t.mt.name)
         serialize(s, collect(Base.MethodList(t.mt)))
         serialize(s, t.mt.max_args)
-        if isdefined(t.mt, :kwsorter)
-            serialize(s, t.mt.kwsorter)
-        else
-            writetag(s.io, UNDEFREF_TAG)
-        end
+        writetag(s.io, UNDEFREF_TAG)
```

Comment:
Seems to me we will have to look up the needed methods of `kwcall` and serialize them as well, otherwise sending functions with keyword arguments won't work.

Reply:
that seems somewhat undefined behavior either way, but possibly true

Reply:
I wrote a test, that worked on master, and didn't work here, so this is fixed now. I don't know if it works well in general (either before or after this change).